xernel/kernel/src/mem/paging.rs

```diff
@@ -293,11 +293,79 @@ impl Pagemap {
             (*pml1)[virt.p1_index()].set_unused();
         }
     }
+
+    /// Only works with 4KiB pages
```

Comment:
Possible to add support for greater page sizes? If yes, do so

Reply:
Possible, but not easy
This will be done in the future